core/src/main/java/hudson/model/Run.java

```diff
@@ -41,6 +41,8 @@
 import hudson.console.ConsoleNote;
 import hudson.console.ModelHyperlinkNote;
 import hudson.console.PlainTextConsoleOutputStream;
+
+import java.io.*;
```

Comment:
I don't think we should use `*` imports on production code. Could you please revert that? Thanks.

Reply:
That's just the IDE trying to bring a bit of order, but ok - I reverted that. 
Is there a style guide published anywhere? Or a "Recommended IDE settings" page that I could go through? The conventions get hard to follow very quickly as different parts of the code base seem to hold to different standards.